### PR TITLE
Fixing typos - readthedocs and plugin_setup.py

### PR DESCRIPTION
**Pull request by vitorheidrich**
Hi,

Today I read 80% of the documentation available for q2-sidle. Really nice job! I intend to use it in the coming weeks.
Because I noticed some typos in the docs, especially throughout the command line functions/parameters descriptions, I decided to fix the ones I spotted. Not sure if you are interested, but I hope this helps.

Thank you very much

### CODE REVIEW

File: q2_sidle/plugin_setup.py [hunk 1]
```diff
@@ -245,81 +245,81 @@
...
-                              "sequences before degenerate expansion should "
-                              "be used."),
+        'region': ('The name of the sub region used in alignment. The region '
+                   'names do not matter, however, the region order must '
+                   'match the order along the hypervariable regions.'),
+        'count_degenerates': ('Whether sequences which contain degenerate '
+                              'nucleotides should be counted as unqiue kmers '
+                              'or whether the number of original database '
+                              'sequences before degenerate expansion should '
+                              'be used."),
```
Comment (jwdebelius) [inline q2_sidle/plugin_setup.py:284]: This line isn't closed dproperly. It either needs to endd with a `'` or with a `"`, but either way, it needs to match.

**Comment by jwdebelius** [comment]
Hi @vitorheidrich,

Thanks! I'm sorry I didn't see this sooner, I'm not sure what's up with my notifications on this repo. Typos are like wack-a-mole for me - I can write code, but I can't see spelling mistaakes.

There's a major single line issue I commented on - can you please fix that and we can make sure the tests pass before merge.

Thank you!

**Comment by vitorheidrich** [comment]
Hi,

Sorry I missed that. Last time I checked the checks were in progress. Hope it's alright now.

Thank you,

**Comment by jwdebelius** [comment]
Thank you so much @vitorheidrich!

**Pull request merged**